src/blog/article/BlogArticlePage.tsx

```diff
@@ -26,77 +28,124 @@ const BlogArticlePage: React.FC<BlogArticlePageProps> = ({
       title,
       content: { json },
       author,
+      updatedAt,
     },
     otherContentfulBlogArticles: { edges },
   },
   location: { href },
-}) => (
-  <>
-    <RippedSection rips={{ bottom: { color: 'white' } }}>
-      <Image
-        fluid={getImageForBreakpoint(hero)}
-        style={{ width: '100%' }}
-        aria-hidden="true"
+}) => {
+  const orgName = 'Pride in London'
+  const firstParagraph = getFirstParagraph(json)
+  const image = `https:${hero.tablet.src}`
+  return (
+    <>
+      <Helmet
+        title={title}
+        siteUrl={href}
+        description={firstParagraph}
+        img={image}
       />
-    </RippedSection>
-    <Wrapper
-      display="flex"
-      justifyContent="center"
-      marginTop="xxl"
-      marginBottom="lg"
-    >
-      <Tag color={getCategoryColor(category)}>{category}</Tag>
-    </Wrapper>
-    <Wrapper
-      display="flex"
-      flexDirection="column"
-      alignItems="center"
-      justifyContent="center"
-      paddingX={{ default: 'lg', md: 'xxl' }}
-      position="relative"
-    >
-      <H2 as="h1" textAlign="center" maxWidth={842}>
-        {title}
-      </H2>
+      <ReactHelmet
+        script={[
+          {
+            '@context': 'https://schema.org',
+            '@type': 'BlogPosting',
+            mainEntityOfPage: {
+              '@type': 'WebPage',
+              '@id': href,
+            },
+            headline: title,
+            image: {
+              '@type': 'ImageObject',
+              url: hero,
+            },
+            dateModified: updatedAt,
+            author: author
+              ? {
+                  '@type': 'Person',
+                  name: author.name,
+                }
+              : { '@type': 'Organization', orgName },
+            publisher: {
+              '@type': 'Organization',
+              orgName,
+              logo: {
+                '@type': 'ImageObject',
+                url: `${href}/logo.png`,
+                // recommended ratio 1.91:1
+                height: 502,
+                width: 960,
+              },
+            },
+          },
+        ]}
+      />
```

Comment:
Can we pop the helmet in a new file, say `BlogArticlePageHelmet.tsx`? 🙏 similar to how we have for `YouMeUsWePage`.

Reply:
Means a clearer separation between the visual page and the metadata/scripts.